server/routes/intention-for-item.route.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const RedisService = require('../services/redis.service')
-const { Paths, RedisKeys, SaleIntention, Views } = require('../utils/constants')
+const { Paths, RedisKeys, Intention, Views } = require('../utils/constants')
 const { buildErrorSummary, Validators } = require('../utils/validation')
 
 const handlers = {
```

Comment:
On lines 10 & 21 there is no need to pass the request to _getContext. Something Sonarcloud picked up

Reply:
Fixed.